docs/articles/faq/README.md

```diff
@@ -283,3 +283,21 @@ According to users' feedback, the following CI systems work best with TestCafe:
 * [TravisCI](../documentation/guides/continuous-integration/travis.md)
 * [CircleCI](../documentation/guides/continuous-integration/circleci.md)
 * [AppVeyor](../documentation/guides/continuous-integration/appveyor.md)
+
+### 'Browser can't open the page: can't establish a secure connection to the server'
+
+If the tested website is served via OpenVPN, TestCafe may be unable to detect an IP address from where to retrieve the website. In this case, the following error occurs:
```

Comment:
> OpenVPN

I thought OpenVPN is a brand, isn't it? Is this issue product-specific?

Reply:
You're right, the issue doesn't seem to be product specific.